vms/shift/templates/shift/edit_hours.html

```diff
@@ -1,6 +1,22 @@
 {% extends "vms/base.html" %}
 
 {% block content %}
+<script type="text/javascript">
```

Comment:
@vubo Adding a separate is a better option. More code will make the things confusing. File inclusion helps when such things happen. 

Do we have any other JS file already there which we can use to input this code?